engine.js

```diff
@@ -31,22 +31,32 @@ module.exports = function(options) {
   var getFromOptionsOrDefaults = function(key) {
     return options[key] || defaults[key];
   };
-  var getJiraIssueLocation = function(location, type, scope, jiraWithDecorators, subject) {
-    switch(location) {
+  var getJiraIssueLocation = function(
+    location,
+    type = '',
+    scope = '',
+    jiraWithDecorators,
+    subject
+  ) {
+    let headerPrefix = type + scope;
+    if (headerPrefix?.length > 0) {
```

Comment:
What do you think about comparing with an empty string? I think that could make it more clear what the comparison is doing.

```suggestion
    if (headerPrefix !== '') {
```

Reply:
Can do.